tests/integration/common_workflows/test_sql_asset_workflows.py

```diff
@@ -30,6 +42,30 @@
 VALUES_ON_MY_FAVORITE_DAY = [8]
 
 
+@pytest.fixture(scope="module", autouse=True)
+def setup_module():
+    # setup table
```

Comment:
These tests rely on a table that used to be created and populated in the test_expectations_v3_api.py file. They would only pass if run after those tests and would fail if run in isolation. I've moved the table creation logic to this module as well as an edited version of that 1 json file that defined this table.